tests/formulas/formulas.test.ts

```diff
@@ -100,3 +106,29 @@ describe("createAdaptedRanges", () => {
     expect(moveFormula(model, "='Sheet 2'!B2", 1, 10)).toEqual("='Sheet 2'!C12");
   });
 });
+
+describe("Remove columns/rows that are references of formula", () => {
+  let model: Model;
+  beforeEach(() => {
+    model = new Model({
+      sheets: [
+        {
+          colNumber: 10,
+          rowNumber: 10,
+        },
+      ],
+    });
+  });
+
+  test("delete multiple columns, including ones in formula", () => {
+    setCellContent(model, "A1", "=C1");
+    deleteColumns(model, ["B", "C"]);
```

Comment:
We should have both tests `delete B,C` and `delete C,D`, because one was working the other not

Reply:
@hokolomopo Deleting C and D will also not work (show #REF error in Google Sheets). Should I test it anyway?

Reply:
Deleting `B,C` and deleting `C,D` should have the same result (=#REF)

The problam was that before your PR delting `C,D` worked, the formula became `=#REF`, but deleting `B,C` didn't work, the formula didn't change. That's why I'm saying that we should test both.